### PR TITLE
[TECHNICAL-SUPPORT] LPS-82215 DynamicQuery pagination

### PR DESCRIPTION
**Pull request by Preston-Crary**

### CODE REVIEW

**Comment by Preston-Crary** [comment]
ci:test

File: portal-impl/test/integration/com/liferay/portal/dao/orm/hibernate/DynamicQueryTest.java [hunk 1]
```diff
@@ -75,8 +82,42 @@ public void testCriterion() {
...
+		DB db = DBManagerUtil.getDB();
+
+		DBType dbType = db.getDBType();
+
+		int databaseInMaxParameters = GetterUtil.getInteger(
+			PropsUtil.get(
+				PropsKeys.DATABASE_IN_MAX_PARAMETERS,
+				new Filter(dbType.getName())));
+
+		Assume.assumeTrue(databaseInMaxParameters > 0);
```
Comment (shuyangzhou) [inline portal-impl/test/integration/com/liferay/portal/dao/orm/hibernate/DynamicQueryTest.java]: @robertoDiaz this assume and https://github.com/shuyangzhou/liferay-portal/pull/6381/commits/1ba580c1066df5c549891b64b7d2b23673a004f5#diff-1fb7d9617dec4fc3e46bffea9f059656R61 are both wrong.
We provide this as a general feature, any customer can turn this on for their DB. 
By default, we only configure this for oracle, which means with these assume we don't test them on any other DBs, which just means we put this out without any tests for non-oracle customers. That is not acceptable.

You need to make these tests run on all dbs we have on CI. Remove those assumes and force set the in max size no matter what db you are using.

**Comment by shuyangzhou** [comment]
@robertoDiaz  see https://github.com/shuyangzhou/liferay-portal/pull/6381/commits/1ba580c1066df5c549891b64b7d2b23673a004f5#r199220828

Please fix and resend.

**Comment by robertoDiaz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**